pages/DynamicPOC/index.tsx

```diff
@@ -63,22 +49,46 @@ const TitleText = styled.h1`
   font-size: large;
 `
 
-const DynamicPOC: React.FC = () => {
+const files = {
+  animalForm: '../../../constants/Animal Form.csv',
+  actualForm: '../../../constants/EdLaw Combined Flowchart.csv',
+}
+
+export const getStaticProps: GetStaticProps = (context) => {
```

Comment:
This is a funky Next.js way of passing props from server side into a component. Some of the modules used for reading the csv are backend only, which means if I try to call the csvToQuestionArray directly in the component it will throw an error. So, the workaround is using getStaticProps which will run this code server-side and then pass the result into DynamicPOC. Here's a link to the docs: https://nextjs.org/docs/basic-features/data-fetching/get-static-props

Reply:
I should clarify, this never gets called explicitly by us, but by exporting it Next knows to use it when building the pages